astropy/stats/funcs.py

```diff
@@ -444,6 +446,193 @@ def true_efficiency(x):
+    lower limits can be badly wrong. The interval is
+
+    .. math::
+
+        CI = (n-1-\sqrt{n+0.75}, n+1+\sqrt{n+0.75})
+
+    **5. 'frequentist-confidence'** These are frequentist central
+    confidence intervals:
+
+    .. math::
+
+        CI = (0.5 F_{\chi^2}^{-1}(\alpha;2n),
+              0.5 F_{\chi^2}^{-1}(1-\alpha;2(n+1)))
+
+    where :math:`F_{\chi^2}^{-1}` is the quantile of the chi-square
+    distribution with the indicated number of degrees of freedom and
+    :math:\alpha is the one-tailed probability of the normal
+    distribution (at the point given by the parameter 'sigma'). See
+    [Maxwell 2011][maxw11] for further details.
+
+    Examples
+    --------
+
+    >>> poisson_conf_interval(np.arange(10), interval='root-n').T
+    array([[  0.        ,   0.        ],
+           [  0.        ,   2.        ],
+           [  0.58578644,   3.41421356],
+           [  1.26794919,   4.73205081],
+           [  2.        ,   6.        ],
+           [  2.76393202,   7.23606798],
+           [  3.55051026,   8.44948974],
+           [  4.35424869,   9.64575131],
+           [  5.17157288,  10.82842712],
+           [  6.        ,  12.        ]])
+
+    >>> poisson_conf_interval(np.arange(10), interval='root-n-0').T
+    array([[  0.        ,   1.        ],
+           [  0.        ,   2.        ],
+           [  0.58578644,   3.41421356],
+           [  1.26794919,   4.73205081],
+           [  2.        ,   6.        ],
+           [  2.76393202,   7.23606798],
+           [  3.55051026,   8.44948974],
+           [  4.35424869,   9.64575131],
+           [  5.17157288,  10.82842712],
+           [  6.        ,  12.        ]])
+
+    >>> poisson_conf_interval(np.arange(10), interval='pearson').T
+    array([[  0.        ,   1.        ],
+           [  0.38196601,   2.61803399],
+           [  1.        ,   4.        ],
+           [  1.69722436,   5.30277564],
+           [  2.43844719,   6.56155281],
+           [  3.20871215,   7.79128785],
+           [  4.        ,   9.        ],
+           [  4.8074176 ,  10.1925824 ],
+           [  5.62771868,  11.37228132],
+           [  6.45861873,  12.54138127]])
+    
+    >>> poisson_conf_interval(np.arange(10),
+    ...                       interval='frequentist-confidence').T
+    array([[  0.        ,   1.84102165],
+           [  0.17275378,   3.29952656],
+           [  0.70818544,   4.63785962],
+           [  1.36729531,   5.91818583],
+           [  2.08566081,   7.16275317],
+           [  2.84030886,   8.38247265],
+           [  3.62006862,   9.58364155],
+           [  4.41852954,  10.77028072],
+           [  5.23161394,  11.94514152],
+           [  6.05653896,  13.11020414]])
+
+    >>> poisson_conf_interval(7,
+    ...                       interval='frequentist-confidence').T
+    array([  4.41852954,  10.77028072])
+                   
+    [pois_eb]: http://www-cdf.fnal.gov/physics/statistics/notes/pois_eb.txt
```

Comment:
@aarchiba - The references in this docstring in the HTML output could look nicer: see [here](http://www.mpi-hd.mpg.de/personalhomes/deil/public/astropy/pr-3717/html/api/astropy.stats.poisson_conf_interval.html#astropy.stats.poisson_conf_interval).
I'm not sure, but I think the best way to mark up references in docstrings is like [this](http://astropy.readthedocs.org/en/latest/api/astropy.stats.binom_conf_interval.html)?